src/utilcode/clrhost_nodependencies.cpp

```diff
@@ -383,6 +394,10 @@ FastFreeInProcessHeapFunc __ClrFreeInProcessHeap = (FastFreeInProcessHeapFunc) C
 
 const NoThrow nothrow = { 0 };
 
+#ifdef HAS_ADDRESS_SANITIZER
+// use standard heap functions for address santizier
```

Comment:
Yes good point. This is an issue when using the sanitizer as it doesn't support the modes that throw an exception. Not sure of the impact (tests?) yet as I need to fix some other issues first. One option is to wrap these and throw an exception ourselves.